rasa_core/channels/socketio.py

```diff
@@ -125,10 +122,9 @@ def __init__(self,
 
     def blueprint(self, on_new_message):
         sio = socketio.Server()
-        socketio_webhook = SocketBlueprint(sio, self.socketio_path,
-                                           'socketio_webhook', __name__)
+        socketio_webhook = SocketBlueprint(sio, self.socketio_path, 'socketio_webhook', __name__)
 
-        @socketio_webhook.route("/", methods=['GET'])
+        @socketio_webhook.route("/health", methods=['GET'])
```

Comment:
the `health` endpoints for all the other channels is at `/` as well, do we need to change this here?